src/client/styles/globals.css

```diff
@@ -195,4 +282,31 @@ table {
   .header-bottom {
     padding: 0 20px;
   }
+}
+
+/* Home Page */
+
+.tour,
+.info {
+  background-image: url(src/client/styles/img/vintageWood.webp);
```

Comment:
it's pretty difficult to see the text on this wood background. perhaps we need to change the font color to make the text more visible.

Reply:
Yes I agree. I want to something behind the text to make it more readable, I think Jerrod was doing it though, so didn't want to add something that would conflict.